app/models/user_session.rb

```diff
@@ -70,4 +72,8 @@ def user_intervention_finished?
 
     user_intervention.completed_sessions == user_intervention.sessions.size
   end
+
+  def intervention_inside_organization?
+    user_intervention&.intervention&.organization_id.present?
```

Comment:
why `&` between `user_intervention` and `intervention`?

Reply:
I have refactored it to include a guard
https://stackoverflow.com/questions/76342090/rails-7-custom-validation-breaks-rspec-model-validations-tests

Reply:
Even if we make a temporary active record object and want to validate it, the validation should return some logical value rather than raising an exception, I believe